es-ds-docs/pages/organisms/cta-card.vue

```diff
@@ -0,0 +1,212 @@
+<script setup>
+const propTableRows = [
+    ['imageFirst', 'Boolean', 'false', 'If true, reorders the elements so the image appears above the heading.'],
+    [
+        'horizontalPadding',
+        'String',
+        "'px-100'",
+        'The padding on the left and right sides of the card. Defaults to 16px.',
+    ],
+    ['verticalSpacing', 'String', "'mb-150'", 'The vertical spacing between elements in the card. Defaults to 24px.'],
+];
+
+const slotTableRows = [
```

Comment:
You can remove the `Type` column here since slots don't have a type. For some prior art, check out /organisms/support-card

Reply:
Nice! Updated.